man/systemd.network.xml

```diff
@@ -391,6 +391,16 @@
           <literal>no</literal>.</para></listitem>
         </varlistentry>
         <varlistentry>
+          <term><varname>IPv6Privacyxtensions=</varname></term>
+          <listitem><para>Configures IPv6 Privacy extentions temporary addresses.
+          Preference for Privacy Extensions (RFC3041).
+          <literal>0</literal> disable Privacy Extensions.
+          <literal>1</literal> enable Privacy Extensions, but prefer public addresses over temporary addresses.
+          greater than <literal>1</literal> enable Privacy Extensions and prefer temporary addresses over public addresses.
+          Defaults to
+          <literal>0</literal>.</para></listitem>
```

Comment:
I'm not sure if exposing this strange kernel-level interface in config files is really our best option. I guess a set of wee-defined strings would be a better match.